l10n_ro_account_report/__openerp__.py

```diff
@@ -19,12 +20,18 @@
 #
 ##############################################################################
 {
-    "name" : "Romania - Account Reports",
-    "version" : "1.0",
-    "author" : "FOREST AND BIOMASS SERVICES ROMANIA	",
+    "name": "Romania - Account Reports",
+    "version": "1.0",
+    "author": "FOREST AND BIOMASS SERVICES ROMANIA	",
     "website": "http://www.forbiom.eu",
-    "category" : "Romania Adaptation",
-    "depends": ['account','report','account_vat_on_payment','l10n_ro_invoice_line_not_deductible','l10n_ro_config'],
+    "category": "Romania Adaptation",
+    "depends": [
+        'account',
+        'report',
+        'account_vat_on_payment',
+        'l10n_ro_invoice_line_not_deductible',
+        'l10n_ro_config'
+    ],
```

Comment:
Mai trebuie si modulul hr sau mai bine e ca in d394 sa prea numele si prenumele din fereastra de wizard